hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/MasterAddressTracker.java

```diff
@@ -297,7 +297,7 @@ public static boolean deleteIfEquals(ZKWatcher zkw, final String content) {
   public static List<ServerName> getBackupMastersAndRenewWatch(
       ZKWatcher zkw) throws InterruptedIOException {
     // Build Set of backup masters from ZK nodes
-    List<String> backupMasterStrings = Collections.emptyList();
```

Comment:
Why change this?

Reply:
There is not need to assign value to `backupMasterStrings` see code below
 
![image](https://user-images.githubusercontent.com/15794564/100176111-a63b1380-2f0a-11eb-9c1c-afd803abc131.png)